metadata-service/restli-client/src/main/java/com/linkedin/common/client/ClientRetry.java

```diff
@@ -0,0 +1,33 @@
+package com.linkedin.common.client;
+
+import com.linkedin.r2.RemoteInvocationException;
+
+import java.util.concurrent.Callable;
+
+public class ClientRetry {
+    protected int getRetries() {
+        return 2;
+    }
+
+    protected int getDelayMs() {
+        return 2000;
+    }
+
+    public <V> V retry(Callable<V> callable) throws RemoteInvocationException {
```

Comment:
Very minor: Is there an existing lib / class we can use to retry a callable?

Reply:
Swapped for some parseq retry classes, this is removed.